templates/includes/benefits.html

```diff
@@ -34,10 +34,20 @@ <h4 class="grid_separator--s" ga-event-category="donations" ga-event-action="ben
 
 <h4 class="grid_separator--s">Giving circles</h4>
 <div class="prose grid_separator--l">
-  <p>Circle members provide sustainable support with three-year commitments to fund our newsroom. Members receive all of the Involved benefits plus access to exclusive events and behind-the-scenes information.</p>
+  <p>Circle members provide sustainable support with three-year commitments to fund our newsroom. Members receive all of the Involved benefits plus access to exclusive events and behind-the-scenes information. <a href="https://stripe-testing.texastribune.org/circleform"> Read more about the Circle Membership </a></p>
```

Comment:
That link is going to the test server

Reply:
Crikey. That was in my notes to self to change before production deploy. Great catch!!!